translate/lang/data.py

```diff
@@ -102,6 +102,7 @@
     'jbo': ('Lojban', 1, '0'),
     'jv': ('Javanese', 2, '(n != 1)'),
     'ka': ('Georgian', 1, '0'),
+    'kab': ('Kabyle', 1, '(n > 1)'),
```

Comment:
According to what we have in some Pootle deployments this should be:

```python
    'kab': ('Kabyle', 2, '(n != 1)'),
```

but I will defer to you to tell.